cpp/src/parquet/file_reader.h

```diff
@@ -56,6 +56,22 @@ class PARQUET_EXPORT RowGroupReader {
   // column. Ownership is shared with the RowGroupReader.
   std::shared_ptr<ColumnReader> Column(int i);
 
+  // Construct a ColumnReader, trying to enable exposed encoding.
+  //
+  // The encoding can only be exposed if the column chunk is encoded with the
```

Comment:
I think this first sentence is redundant and might not be true if future encodings are exposed.  Lets keep the details specific for dictionary.